jobs/datalake-data-export/src/services/__tests__/read-model-queries.service.test.ts

```diff
@@ -96,7 +96,7 @@ describe('read-model-queries.service', () => {
     function toReadModelEServices(
       eservices: Array<EService>
     ): Array<{ data: { id: string; descriptors: Array<{ state: DescriptorState }> } }> {
-      return eservices.map((eservice) => ({ data: eservice }))
+      return eservices.map((eservice) => ({ data: { ...eservice, createdAt: new Date().toISOString() } }))
```

Comment:
does this mean that the `createdAt` is not in the `eservice`?

Reply:
No, It is actually present, the goal was just to convert the createdAt into a string. I created a new one, using the one already present in the e-service is also an option